build/lib/release.sh

```diff
@@ -196,76 +201,26 @@ function kube::release::sha1() {
-  (
-    local binary_dir="$1"
-    local arch="$2"
-    local binary_name
-    local binaries=($(kube::build::get_docker_wrapped_binaries ${arch}))
-
-    for wrappable in "${binaries[@]}"; do
-
-      local oldifs=$IFS
-      IFS=","
-      set $wrappable
-      IFS=$oldifs
-
-      local binary_name="$1"
-      local base_image="$2"
-
-      kube::log::status "Starting Docker build for image: ${binary_name}"
-
-      (
-        local md5_sum
-        md5_sum=$(kube::release::md5 "${binary_dir}/${binary_name}")
-
-        local docker_build_path="${binary_dir}/${binary_name}.dockerbuild"
-        local docker_file_path="${docker_build_path}/Dockerfile"
-        local binary_file_path="${binary_dir}/${binary_name}"
-
-        rm -rf ${docker_build_path}
-        mkdir -p ${docker_build_path}
-        ln ${binary_dir}/${binary_name} ${docker_build_path}/${binary_name}
-        printf " FROM ${base_image} \n ADD ${binary_name} /usr/local/bin/${binary_name}\n" > ${docker_file_path}
-
-        if [[ ${arch} == "amd64" ]]; then
-          # If we are building a amd64 docker image, preserve the original image name
-          local docker_image_tag=gcr.io/google_containers/${binary_name}:${md5_sum}
-        else
-          # If we are building a docker image for another architecture, append the arch in the image tag
-          local docker_image_tag=gcr.io/google_containers/${binary_name}-${arch}:${md5_sum}
-        fi
-
-        "${DOCKER[@]}" build -q -t "${docker_image_tag}" ${docker_build_path} >/dev/null
-        "${DOCKER[@]}" save ${docker_image_tag} > ${binary_dir}/${binary_name}.tar
-        echo $md5_sum > ${binary_dir}/${binary_name}.docker_tag
-
-        rm -rf ${docker_build_path}
-
-        # If we are building an official/alpha/beta release we want to keep docker images
-        # and tag them appropriately.
-        if [[ -n "${KUBE_DOCKER_IMAGE_TAG-}" && -n "${KUBE_DOCKER_REGISTRY-}" ]]; then
-          local release_docker_image_tag="${KUBE_DOCKER_REGISTRY}/${binary_name}-${arch}:${KUBE_DOCKER_IMAGE_TAG}"
-          kube::log::status "Tagging docker image ${docker_image_tag} as ${release_docker_image_tag}"
-          docker rmi "${release_docker_image_tag}" || true
-          "${DOCKER[@]}" tag "${docker_image_tag}" "${release_docker_image_tag}" 2>/dev/null
-        fi
-
-        kube::log::status "Deleting docker image ${docker_image_tag}"
-        "${DOCKER[@]}" rmi ${docker_image_tag} 2>/dev/null || true
-      ) &
-    done
-
-    kube::util::wait-for-jobs || { kube::log::error "previous Docker build failed"; return 1; }
-    kube::log::status "Docker builds done"
-  )
-
+function kube::release::package_hyperkube_for_server() {
+  # We know this was built above in package_hyperkube, just reuse its variables
+  local binary_dir="$1"
+  local arch="$2"
+
+  local tag_arch="-${arch}"
+  if [[ "${arch}" == "amd64" ]]; then
+    tag_arch=""
+    # amd64 is canonically `hyperkube`, not `hyperkube-amd64`
```

Comment:
Thanks! I wasn't sure which one was more canonical, I'll swap them all to the `-amd64` variant.